services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/HealthCheckSpecification.groovy

```diff
@@ -50,5 +50,12 @@ class HealthCheckSpecification extends BaseSpecification {
                 }.isEmpty()
             }.empty
         }
+
+        and: "Multi table is off"
+        verifyAll {
+            topology.activeSwitches.findAll {
+                northbound.getSwitchProperties(it.dpId).multiTable
+            }.empty
+        }
```

Comment:
I'm going to remove this in my PR since in develop branch this behavior is driven by a global property `use.multitable`, so you cannot assume this in health check. Unless you explain why this is required

Reply:
fixed. It was debug part :(